templates/contact-us/index.html

```diff
@@ -0,0 +1,111 @@
+              <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>
+            </li>
+            <li class="p-list__item">
+              <small>In submitting this form, I confirm that I have read and agree to <a href="https://ubuntu.com/legal/data-privacy/contact">Canonical&rsquo;s Privacy Notice</a> and <a href="https://ubuntu.com/legal/data-privacy">Privacy Policy</a>.</small>
+            </li>
+          </ul>
+          <ul class="p-list">
+            <li class="mktField p-list__item">
+              <button type="submit" class="mktoButton p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Form', 'eventAction' : 'cloud contact-us', 'eventLabel' : '{{product}}', 'eventValue' : undefined });">Submit</button>
+            </li>
+          </ul>
+          <!-- hidden fields -->
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formValidation" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" class="mktoField" value="1257" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formVid" class="mktoField" value="1257" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="lpId" class="mktoField" value="2154" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="subId" class="mktoField" value="30" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="munchkinId" class="mktoField" value="066-EOV-335" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="lpurl" class="mktoField" value="https://pages.ubuntu.com/Cloud-contact-us.html?cr={creative}&amp;kw={keyword}" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="cr" class="mktoField" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="kw" class="mktoField" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="q" class="mktoField" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="https://charmed-osm.com/contact-us/thank-you" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="retURL" value="https://charmed-osm.com/contact-us/thank-you" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="return_url" value="https://charmed-osm.com/contact-us/thank-you" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Consent_to_Processing__c" value="yes" />
+        </fieldset>
+      </form>
+    </div>
+
+    <div class="col-4">
+      <div class="p-card">
+        <h3 class="p-card__title">Any questions? Call us</h3>
```

Comment:
If that was true, there would be no point to the contact form, because you’re supposed to call us instead.

```suggestion
        <h2 class="p-card__title">Prefer to call us?</h2>
```